### DISCUSSION TITLE
Proposal: Improve readability for tiddlers, that contain headings with a little bit of CSS

### DISCUSSION THREAD

**Discussion by pmario** [TiddlyWiki/TiddlyWiki5#5285] [General]
I'd like to propose some very subtle CSS changes, that imo greatly improve the readability for tiddlers that contain headings. 

I'd be willing to add a class-field to all tiddlers that need it, without changing the "modified" date. 

To reproduce:

 - Open this link: [Some documentation tiddlers, which are hard to read](https://tiddlywiki.com/prerelease/#:[[Introduction%20to%20filter%20notation]]%20[[Discover%20TiddlyWiki]]%20[[Formatting%20List%20Results%20as%20Tables%20(no%20CSS)]]%20wltc-docs-styles%20[[Formatting%20List%20Results%20as%20Tables%20with%20CSS%20-%20Specified%20Columns%20Methods]])
 - Now extract and import: [improve-readability.zip](https://github.com/Jermolene/TiddlyWiki5/files/5697027/improve-readability.zip)
 - See the difference. 

The following screenshots only give a little glimpse. If you test with the JSON and the prerelease link you'll see what I want. 

<details>
    <summary>Before</summary>

![before](https://user-images.githubusercontent.com/374655/102239725-7377b000-3ef7-11eb-92fd-b0b549815c6b.gif)

</details>

<details>
    <summary>After</summary>

![after](https://user-images.githubusercontent.com/374655/102239744-78d4fa80-3ef7-11eb-9f56-04c08cfb859a.gif)

</details>

**Comment by Jermolene** [comment]
Hi @pmario I think the idea is to use left margins (and padding) on everything except headings to move them to the right, presumably to make the headings easier to distinguish from the text. I think that it breaks an important typographic principle: maintaining a flush left margin. I've never seen a website that does this.

Perhaps a simpler approach is to increase the font-weight of the headings. I keep meaning to make a PR:

<img width="1507" alt="image" src="https://user-images.githubusercontent.com/174761/102241533-fd704a80-3ef0-11eb-93a0-a024c8227643.png">

**Comment by pmario** [comment]
> I've never seen a website that does this.

For me a tiddler is more like a "book page" than a "site page". font-weight is an improvement, to start with.

**Reply by Jermolene** [reply]
That's a reasonable perspective, and I can certainly imagine it being a useful technique to experiment, but I don't think it's universal enough for the core.
